src/Form/ArticleFormType.php

```diff
@@ -37,7 +38,19 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             )
             ->add('name', TextType::class, array('label' => 'Название'))
             ->add('description', TextareaType::class, array('label' => 'Текст'))
-            ->add('save', SubmitType::class, array('label' => 'Сохранить'))
-        ;
+            //            todo: actual demands for image attachment
```

Comment:
@daria-sieroshtan 
looks like you need to rm todo

Reply:
Not yet

Reply:
not yet